tests/postgres_test.py

```diff
@@ -1,12 +1,21 @@
+from unittest.mock import call
+
+import pytest
 import sqlalchemy
 
-CREDENTIALS = dict(user="dbt_sugar_test_user", password="magical_password", database="dbt_sugar")
+from dbt_sugar.core.connectors.postgres_connector import PostgresConnector
+
+CREDENTIALS = {
+    "username": "dbt_sugar_test_user",
+    "password": "magical_password",
+    "database": "dbt_sugar",
```

Comment:
Ok I think I know what is making the test fail, you're not providing some of the required info for a postgress db.
We need a host and a port number. In theory the credentials validation would complain but since you're feeding it directly into the connector rather than via the dbt profile parsing we kind of skip this in the test.

I had a similar problem before.